azure-pipelines.yml

```diff
@@ -267,14 +267,11 @@ jobs:
       # copy the rom to a specific location
       ROM_TARGET="${BIN_DIR}/sw/device/silicon_creator/rom"
       mkdir -p "${ROM_TARGET}"
-      AUTHENTICITY=(fake real)
-      for authenticity in "${AUTHENTICITY[@]}"; do
-        ROM=$(ci/scripts/target-location.sh "//sw/device/silicon_creator/rom:rom_with_${authenticity}_keys_fpga_cw310_scr_vmem")
-        cp $ROM "${ROM_TARGET}/$(basename "${ROM}")"
-        ROM_DIR=$(dirname $ROM)
-        cp "${ROM_DIR}/rom_with_${authenticity}_keys_fpga_cw310.elf" "${ROM_TARGET}/"
-        cp "${ROM_DIR}/rom_with_${authenticity}_keys_fpga_cw310.bin" "${ROM_TARGET}/"
-      done
+      # The deps 2 levels down from the opentitan_rom_binary include all the
+      # important files.
```

Comment:
Nit: can we put this sentence on the same line?

Reply:
Er... do we not attempt to do line limits in these files for readability? I know I didn't try very hard with some of the bash lines, but...

Reply:
I guess it's ok, just looks weird to me when there are other lines that are not wrapped. CI does not seem to mind either way, so I guess this need not be blocking.